lesson01/java.js

```diff
@@ -0,0 +1,105 @@
+/*jQuery document.ready*/
```

Comment:
Default name for Javascript files is always script.js. "Java" is a different programming language with an annoyingly similar name, so we try to avoid shortening "javascript" in that way to avoid confusion.

Reply:
Got it, thank you.

Reply:
> These files should be in the `https://github.com/jocjoh/html300/tree/master/lesson01/assignments` directory. They can just replace the index.html and styles.js files that are already there.

Ok, thanks. Do I need to make this change now for you to grade my assignment or is this for future reference?

Reply:
It's not necessary for this assignment, but do change it for lesson 2 and it may solve the Gulp issues.